docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - mailhog
     restart: always
     command: php -t /srv -S 0.0.0.0:80 .ht.router.php
+#    command: php -d zend_extension=xdebug.so -t /srv -S 0.0.0.0:80 .ht.router.php
```

Comment:
@andypost @jbguerraz  Lets move docker-compose.yml, *.sh and *.make.yml and all environments related  files to separate repo. And put references to proper images from docker hub.

In this case we will have more clear and simpler support process